gateway/webhook/webhook.go

```diff
@@ -247,6 +248,22 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 				continue
 			}
 
+			if misc.ContainsString(sourceListForParsingParams, strings.ToLower(breq.sourceType)) {
+				queryParams := req.request.URL.Query()
+				paramsBytes, err := json.Marshal(queryParams)
+
+				if err != nil {
+					req.done <- webhookErrorRespT{err: response.GetStatus(response.ErrorInMarshal)}
+					continue
+				}
+
+				closingBraceIdx := bytes.LastIndexByte(body, '}')
+				appendData := []byte(`, "query_parameters": ` + string(paramsBytes))
```

Comment:
Can we append paramsBytes to `[]byte(`, "query_parameters":` , Probably skip the conversion to string and back to byte array